modguard/init.py

```diff
@@ -103,4 +106,8 @@ def init_project(root: str, exclude_paths: Optional[list[str]] = None):
             elif write_op.operation == WriteOperation.PUBLIC:
                 mark_as_public(write_op.location, write_op.member_name)
         except errors.ModguardError:
-            print(f"Error marking {write_op.operation} in {write_op.location}")
+            print(
+                f"Error marking {write_op.operation.value}"
+                f"{'({member})'.format(member=write_op.member_name) if write_op.member_name else ''}"
```

Comment:
never seen a format call inside an f string lol might be worth doing this formatting outside of the print() call